### ISSUE TITLE
WGLMakie `DataInspector` is missing some font characters

### ISSUE THREAD

**Issue by kbarros** [MakieOrg/Makie.jl#3516]
**Updated with simplifications**

WGLMakie fails to show certain characters in the DataInspector. Simple example:

```julia
using WGLMakie

labels = ["hello world", "'A' = 2/3"]
inspector_label(_plot, index, _position) = labels[index]
fig = meshscatter(Vec3f.([[0, 0, 0], [1, 0, 0]]); inspector_label)

# The characters `'` and `/` and `=` are all missing. Inspection
# shows " A  = 2 3" instead of "'A'  = 2/3"
DataInspector()
```

<img width="287" alt="image" src="https://github.com/MakieOrg/Makie.jl/assets/290205/c3ca3bd6-68a9-49e2-b7c5-bab9ea216d93">

The problem is specific to the DataInspector. The same characters are available for the `text!` command:

```julia
text("'A' = 2/3") # no problem showing this string
```

**Comment by kbarros** [comment]
This problem is not specific to custom font loading, and also arises for the built-in fonts. MWE:

```julia
using WGLMakie

labels = ["hello world", "'A' = 2/3"]
inspector_label(_plot, index, _position) = labels[index]
fig = meshscatter(Vec3f.([[0, 0, 0], [1, 0, 0]]); inspector_label)

# At this point, the font for characters `'` and `/` is missing. Inspecting the second object
# will print " A  = 2 3" instead of "'A'  = 2/3"
DataInspector()

# This command loads the characters `'` and `/`, fixing the inspector.
text!(fig.axis, Vec3f(0.5, 0.5, 0.5); text="'B' = 6/7")
```

The capture below is taken _before_ calling `text!`.

<img width="367" alt="image" src="https://github.com/MakieOrg/Makie.jl/assets/290205/1ca0ec3a-7e62-4e3b-8d4f-2da1f0d63e99">

**Comment by kbarros** [comment]
Regarding a possible workaround, SimonDanisch suggested:

> I think maybe rendering the glyphs you need before any plot!
E.g. like this function: https://github.com/MakieOrg/Makie.jl/blob/master/src/utilities/texture_atlas.jl#L219-L232

**Comment by ffreyer** [comment]
This will also be fixed by #3737 in the next release

**Comment by kbarros** [comment]
Fantastic! This works for me on #master.

**Comment by ffreyer** [comment]
Should be fixed in 0.20.9+
